### DISCUSSION TITLE
HandyControl + Prism + NLog 支撑 WPF开发常用吗?

### DISCUSSION THREAD

**Discussion by blueyms** [BYJRK/DotNet-Discussions#275] [WPF 开发]
之前做web开发较多，现在转行做WPF，请问HandyControl + Prism + NLog 支撑 WPF开发常用吗，欢迎大家给点选型意见

**Comment by BYJRK** [comment]
这取决于你想做什么类型、什么方向的开发。我个人比较倾向于使用 CommunityToolkit.Mvvm 代替 Prism、使用 Serilog 代替 NLog，主题和控件库则视情况而定

**Reply by blueyms** [reply]
我们这边主要做一些视觉工站系统

**Reply by ZhuYajunFly** [reply]
> 这取决于你想做什么类型、什么方向的开发。我个人比较倾向于使用 CommunityToolkit.Mvvm 代替 Prism、使用 Serilog 代替 NLog，主题和控件库则视情况而定

大佬有Serilog的入门视频不。大佬的视频非常棒。

**Comment by ZhuYajunFly** [comment]
MVVMToolKit + HandyControl + SunnyUI + TouchSocket 就能满足我的日常开发了
